assets/javascripts/discourse/initializers/setup-subscriptions.js

```diff
@@ -23,6 +26,36 @@ export default {
           href: `/u/${user.username}/billing/subscriptions`,
           content: "Billing",
         });
+
+        if (user.admin) {
+          api.modifyClassStatic("model:site-setting", {
+            pluginId: "discourse-subscriptions",
+            update(key, value, opts = {}) {
+              if (key === "discourse_subscriptions_pricing_table") {
```

Comment:
This feels a little fragile... since we control the site setting, could we have the price table id and the publishable key split into separate properties server-side?